### ISSUE TITLE
[FR] Add "live preview" of custom URLs with Twig expressions in Content SEO

### ISSUE THREAD

**Issue by Mosnar** [nystudio107/craft-seomatic#608]
Building out multiple-field based SEO titles, descriptions, and image URLs feels a bit like trial-and-error right now, especially as you start having to scroll through Twig in a full-width, single line, text  input. I'm suggesting adding a "live-preview" of sorts for your current expression, sort of like how Commerce 3's new tax condition tester works. A random entry in the content SEO could be picked and matched against to show you the result of your expression.

**Comment by khalwat** [comment]
Yeah this is something I wanted after I architected how everything worked. And it actually ends up being really hard to do currently.
